lib/railway_ipc/rabbitmq/payload.rb

```diff
@@ -2,6 +2,7 @@
 
 module RailwayIpc
   module Rabbitmq
+    # Why is this class namespaced under Rabbitmq?
```

Comment:
I think this comment can be RM'ed for the PR.

Reply:
yeah I'll remove it